homeassistant/components/light/services.yaml

```diff
@@ -2,13 +2,19 @@
 
 turn_on:
   description: Turn a light on.
+  target:
   fields:
-    entity_id:
-      description: Name(s) of entities to turn on
-      example: "light.kitchen"
     transition:
+      name: Transition
       description: Duration in seconds it takes to get to next state
       example: 60
+      selector:
+        number:
+          min: 0
+          max: 60
```

Comment:
This is the only one I doubt about... 60 second, I don't understand where this max/limit comes from.